ldap_auth_provider.py

```diff
@@ -173,8 +173,10 @@ def check_password(self, user_id, password):
                     ]
                 )
 
-                if len(conn.response) == 1:
-                    attrs = conn.response[0]['attributes']
+                response = [r for r in conn.response if r['type'] == 'searchResEntry']
```

Comment:
Don't use single character variables, doesn't make for a good read in my opinion.
```
responses = [response for response in conn.response if response['type'] == 'searchResEntry']

if len(responses) == 1:
```